client/pyroclient/client.py

```diff
@@ -157,7 +163,14 @@ def create_no_alert_site(self, lat: float, lon: float, name: str, country: str,
                    "geocode": geocode}
         return requests.post(self.routes["no-alert-site"], headers=self.headers, json=payload)
 
-    def send_alert(self, lat: float, lon: float, event_id: int, device_id: int, media_id: int = None) -> Response:
+    def send_alert(
+        self,
+        lat: float,
+        lon: float,
+        device_id: int,
+        event_id: Optional[int] = None,
+        media_id: Optional[int] = None
```

Comment:
By setting media_id optional, an alert can be sent without related media ? (I am just wandering if its related to this PR but why not ! )

Reply:
Oh no, this is something that we decided a long time ago :sweat_smile: (cf. the SQL table field ;) )
It's for the future case when someone notifies an alert but has no mean to send the picture